pkg/components/components.go

```diff
@@ -565,9 +560,19 @@ func GetClusterPermissions() []rbacv1.PolicyRule {
 		roleWithAllPermissions("console.openshift.io", stringListToSlice("consoleclidownloads", "consolequickstarts")),
 		{
 			APIGroups: stringListToSlice(configOpenshiftIO),
-			Resources: stringListToSlice("clusterversions", "infrastructures", "ingresses", "networks"),
+			Resources: stringListToSlice("clusterversions", "infrastructures", "networks"),
 			Verbs:     stringListToSlice("get", "list"),
 		},
+		{
+			APIGroups: stringListToSlice(configOpenshiftIO),
+			Resources: stringListToSlice("ingresses"),
+			Verbs:     stringListToSlice("get", "list", "watch"),
+		},
+		{
+			APIGroups: stringListToSlice(configOpenshiftIO),
+			Resources: stringListToSlice("ingresses/status"),
+			Verbs:     stringListToSlice("update"),
+		},
```

Comment:
Not strictly needed, but maybe implementing a Validating Admission Policy (GA in k8s 1.30) to have an external mechanism as a safety guard to enforce that HCO can only touch a specific subset of the status could be a nice to have addition (even in a subsequent PR).

Reply:
I will look into it. Thanks

Reply:
Not sure it's the right mechanism for that. It's *may* be possible, but I think it's too complicated and easy to write a working CEL expression to do that.